### PR TITLE
add k3d

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

PTAL @iwilltry42

### CODE REVIEW

**Review by iwilltry42 — DISMISSED** [review]
LGTM
@developer-guy thanks for adding k3d here :)

**Comment by developer-guy** [comment]
@iwilltry42 it looks like we are having an error like this: `"go: parsing $GOFLAGS: non-flag "readonly"` and it seems that it didn't parse the `-mod readonly` syntax defined in GOFLAGS environment variable, I think we should update it in [Makefile](https://github.com/k3d-io/k3d/blob/64ba96a25c9531771525ed14b277b8d6ca487484/Makefile#L75) by simply editing it with `-mod=readonly`, WDYT? 

Btw, I tried locally, everything worked fine, do you have any suggestions @kaniini @rawlingsj ?

**Comment by iwilltry42** [comment]
@developer-guy changed in Makefile in `main`. Will be in the next release archive then.

**Comment by developer-guy** [comment]
yep, I saw: https://github.com/k3d-io/k3d/commit/8785f31db152a97ec710a4a2e3abaa6d8b8216ba
great!

**Comment by developer-guy** [comment]
will you also bump a new release?

**Comment by developer-guy** [comment]
kindly ping @iwilltry42

**Comment by iwilltry42** [comment]
Yeah, I'm waiting for Go 1.20 and another PR to build properly.
(At first I was waiting for brew to get 1.20 out 😬)

**Comment by developer-guy** [comment]
is there any update on this? kindly ping @iwilltry42

File: k3d.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  name: k3d
+  version: 5.4.8
+  epoch: 0
+  description: Little helper to run CNCF's k3s in Docker
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
```
Comment (imjasonh) [inline k3d.yaml:11]:
```suggestion
```

File: k3d.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package:
+  name: k3d
+  version: 5.4.8
+  epoch: 0
+  description: Little helper to run CNCF's k3s in Docker
+  target-architecture:
+    - all
```
Comment (imjasonh) [inline k3d.yaml:7]:
```suggestion
```

**Review by imjasonh — DISMISSED** [review]

**Review by joshrwolf — APPROVED** [review]

**Pull request merged**